spec/rails_helper.rb

```diff
@@ -1,27 +1,23 @@
 require "coveralls"
 Coveralls.wear_merged!('rails')
-require 'spec_helper'
+
 ENV['RAILS_ENV'] ||= 'test'
 require File.expand_path('../config/environment', __dir__)
+
 abort("The Rails environment is running in production mode!") if Rails.env.production?
+require 'spec_helper'
 require 'rspec/rails'
 
-begin
-  ActiveRecord::Migration.maintain_test_schema!
-rescue ActiveRecord::PendingMigrationError => e
-  puts e.to_s.strip
-  exit 1
-end
+ActiveRecord::Migration.maintain_test_schema!
 
```

Comment:
Why have you remove this?

Reply:
we thought it was something regarding stripe. now it is back again